src/main/java/com/minecolonies/coremod/event/EventHandler.java

```diff
@@ -420,11 +421,7 @@ public static void onPlayerEnterWorld(final PlayerEvent.PlayerLoggedInEvent even
                 }
             }
 
-            MinecraftServer server = ServerLifecycleHooks.getCurrentServer();
-            if(server instanceof DedicatedServer)
```

Comment:
If this here was a problem, I'm quite sure you can do player.world.isRemote

Reply:
Even on dedicated servers (where all players are remote), that always returns false.  Apparently intentionally, despite what one might expect.

It's not an issue, just philosophically bugged me.

Reply:
No, that's not true, on dedicated servers, there is a "ServerPlayerEntity" and a "ClientPlayerEntity", so depending on the site it's exactly what you want

Reply:
If it returns always false in this case, this means this code is only called on the server side (still makes sense to do a !player.world.isRemote check)

Reply:
This is already wrapped in `if (event.getPlayer() instanceof ServerPlayerEntity)`.  I can add the additional check if you want it for code cleanliness purposes.

Reply:
Then it's not needed at all

Reply:
Aye.  So it's gone now.
The point was to try and avoid sending the unnecessary message for situations where the client and server shared JVM, rather than sidedness, but if that's not necessary or premature optimization, there's no reason to bother.